### PR TITLE
add headStyle to card component

### PR DESCRIPTION
**Pull request by emersonlaurentino**
solving the issue #6654

### CODE REVIEW

**Comment by afc163** [comment]
`feature-3.8.0` branch, please~

**Comment by emersonlaurentino** [comment]
@afc163 changed to branch `feature-3.8.0`.

File: components/drawer/__tests__/__snapshots__/Drawer.test.js.snap [hunk 1]
```diff
@@ -40,7 +39,6 @@ exports[`Drawer destroyOnClose is true 1`] = `
     />
     <div
       class="ant-drawer-content-wrapper"
-      style="transform: translateX(100%);"
```
Comment (afc163) [inline components/drawer/__tests__/__snapshots__/Drawer.test.js.snap]: Just rebase again to remove these changes

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "rc-checkbox": "~2.1.5",
     "rc-collapse": "~1.9.0",
     "rc-dialog": "~7.1.0",
-    "rc-drawer": "~1.5.7",
+    "rc-drawer": "~1.5.9",
```
Comment (afc163) [inline package.json]: inrelevent change

File: components/drawer/__tests__/__snapshots__/Drawer.test.js.snap (hunk 1 above)
Comment (emersonlaurentino) [reply components/drawer/__tests__/__snapshots__/Drawer.test.js.snap]: sorry @afc163 how do I do this?

Comment (afc163) [reply components/drawer/__tests__/__snapshots__/Drawer.test.js.snap]: rebase feature-3.8.0

Comment (emersonlaurentino) [reply components/drawer/__tests__/__snapshots__/Drawer.test.js.snap]: I think now it has.

**Comment by afc163** [comment]
image

**Comment by afc163** [comment]
Maybe you should create a new PR

**Comment by emersonlaurentino** [comment]
okay, I'll do it.

**Comment by emersonlaurentino** [comment]
new pr #11403

**Pull request closed without merging**